navjson/navjson_test.go

```diff
@@ -0,0 +1,182 @@
+package navjson
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParse(t *testing.T) {
+	t.Run("full valid doc", func(t *testing.T) {
+		j, err := Parse(`
+			{
+				"root": "guide/attest/2.7-experiment/",
```

Comment:
use this https://github.com/dequelabs/attest-node-suite/blob/develop/docs/nav.json as test sample.

Reply:
also this one please - https://github.com/dequelabs/attest-docs/blob/master/docs/nav.json

Reply:
Done! See `TestParseExamples` fn below